ledger/accountdb_test.go

```diff
@@ -662,19 +662,14 @@ func randomCreatable(uniqueAssetIds map[basics.CreatableIndex]bool) (
 	return assetIdx, creatable
 }
 
-func BenchmarkReadingAllBalances(b *testing.B) {
-	proto := config.Consensus[protocol.ConsensusCurrentVersion]
-	//b.N = 50000
-	dbs, _ := dbOpenTest(b, true)
-	setDbLogging(b, dbs)
-	defer dbs.close()
-
+func benchmarkInitBalances(b *testing.B, dbs dbPair, proto config.ConsensusParams) (updates map[basics.Address]basics.AccountData) {
```

Comment:
I think it would make it much more flexible if you were to change the function signature to receive a `testing.TB` instead of `testing.B` and add the accounts number as the argument.